src/router/index.ts

```diff
@@ -26,4 +43,14 @@ const router = new VueRouter({
   routes,
 });
 
+router.beforeEach((to, from, next) => {
+  if (to.name === null) {
+    next('/');
+  } else if (to.meta.requiresAuth && storeModule.userId === '') {
```

Comment:
Change condition to work with const and with the changes in the storeModule as mentioned.

Reply:
Can you explain again?

Reply:
Like reducer initialize with react, using a const that is equal to the initial state, and use this const in line 49, compare to a const not ‘’